examples/gallery/webllm.ipynb

```diff
@@ -34,9 +34,9 @@
     "MODELS = {\n",
     "    'SmolLM (130MB)': 'SmolLM-135M-Instruct-q4f16_1-MLC',\n",
     "    'TinyLlama-1.1B-Chat (675 MB)': 'TinyLlama-1.1B-Chat-v1.0-q4f16_1-MLC-1k',\n",
-    "    'Gemma-2b (1895 MB)': 'gemma-2-2b-it-q4f16_1-MLC',\n",
-    "    'Mistral-7b-Instruct (4570 MB)': 'Mistral-7B-Instruct-v0.3-q4f16_1-MLC',\n",
-    "    'Llama-3.1-8b-Instruct (4598 MB)': 'Llama-3-8B-Instruct-q4f16_1-MLC-1k',\n",
+    "    'Gemma-2b (2GB)': 'gemma-2-2b-it-q4f16_1-MLC',\n",
+    "    'Mistral-7b-Instruct (5GB)': 'Mistral-7B-Instruct-v0.3-q4f16_1-MLC',\n",
+    "    'Llama-3.2-1B-Instruct (5GB)': 'Llama-3.2-1B-Instruct-q4f16_1-MLC',\n",
```

Comment:
I don't think the 3.2 billion model is 5 GB.

Reply:
Oh yeah, this isn't right at all.